src/scanoss/spdxlite.py

```
@@ -185,7 +185,12 @@ def produce_from_json(self, data: json, output_file: str = None) -> bool:
             'name': 'SCANOSS-SBOM',
             'creationInfo': {
                 'created': now.strftime('%Y-%m-%dT%H:%M:%SZ'),
-                'creators': [f'Tool: SCANOSS-PY: {__version__}', f'Person: {getpass.getuser()}'],
+                'creators': [
+                    f'Tool: SCANOSS-PY: {__version__}',
+                    f'Person: {getpass.getuser()}',
+                    f'Organization: SCANOSS'
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)